clients/extension/src/pages/transaction/index.tsx

```diff
@@ -16,10 +16,7 @@ import {
   SquareBehindSquare,
 } from '@clients/extension/src/icons'
 import api from '@clients/extension/src/utils/api'
-import {
-  explorerUrl,
-  TssKeysignType,
-} from '@clients/extension/src/utils/constants'
+import { TssKeysignType } from '@clients/extension/src/utils/constants'
```

Comment:
we have the same definition call `SignatureAlgorithm` in `core` package as well, consider to use that one instead

Consider to update it in a seperate PR